### PR TITLE
HZC-4932: Make label optional for some HIVE components

### PR DESCRIPTION
**Pull request by yaya-usman**
[HZC-4932](https://hazelcast.atlassian.net/browse/HZC-4932)

[HZC-4932]: https://hazelcast.atlassian.net/browse/HZC-4932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by allfayn** [comment]
Could we add components without labels to storybook stories?

**Comment by yaya-usman** [comment]
@allfayn Yes we can, confirmed that already

File: packages/ui/src/Toggle.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ type ToggleCoreProps = {
 }
 
 export type ToggleExtraProps = {
-  label: string | React.ReactNode
+  label?: string | React.ReactNode
```
[the remark this answers is not in the document]
Comment (fantkolja) [reply packages/ui/src/Toggle.tsx:21]: Check other components

**Review by fantkolja — CHANGES_REQUESTED** [review]

File: packages/ui/src/Toggle.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (yaya-usman) [reply packages/ui/src/Toggle.tsx:21]: i think conditional rendering is not necessary in this case actually, i have tested this on storybook, it's not having any issue likewise all the components i made changes to. i have removed the label and included it to check the changes it works just fine.

**Pull request merged**